vue-frontend/src/assets/css/tailwind.css

```diff
@@ -19,7 +19,27 @@
   font-display: swap;
   src: url(./../fonts/FuturaLT-Bold.woff2);
 }
-
+@font-face {
+  font-family: "Inter-Bold";
```

Comment:
I think there is no `Inter-Bold` text downloaded. Also `SF-pro` and `Roboto` text have not been used anywhere. Why you have downloaded it?